Automattic/woocommerce-payments
# Allow test support phone in dev mode

Fixes #6782 

#### Changes proposed in this Pull Request

- Checks for dev mode and allows a test phone number i.e. +1 000-000-0000

#### Testing instructions

- In dev mode, check that you can save all zeros number (allowed by Stripe for test accounts)

-------------------


**Post merge**
